src/main/java/com/google/javascript/gents/CommentLinkingPass.java

```diff
@@ -33,8 +33,11 @@
    */
   private static final Pattern[] JSDOC_REPLACEMENTS = {
     Pattern.compile(
-        BEGIN_JSDOC_LINE + "@(extends|implements|type)[ \t]*(\\{[^@]*\\})[ \t]*(?<keep>)" + EOL,
+        BEGIN_JSDOC_LINE + "@(implements|type)[ \t]*(\\{[^@]*\\})[ \t]*(?<keep>)" + EOL,
         Pattern.DOTALL),
+    //keep generic extends for post processing
```

Comment:
add a space after '//' and '// TODO: investigate whether generics params can be emitted correctly'.